### PR TITLE
First draft of ACCESSIBILITY.md

### LINKED ISSUE #3145: Create ACCESSIBILITY.md file - accessibility policy for the community - drawing from the Slack policies: https://hackmd.io/@turingway/ttw-slack-intro

### PR DESCRIPTION
**Pull request by aleesteele**
### Summary

Fixes #3145 

This draft is incomplete. It has information as it relates to accessibility across different aspects of the project (and how we as a community would like to promote accessibility!), and builds upon @AndreaSanchezTapia and @LizHareDogs as originally drafted for the upcoming guide for accessibility alongside @malvikasharan and others (documented in #2730 & #2942) and @AlexandraAAJ's draft about accessibility for events, as well as the translation & localisation's teams understanding of accessibility as a socio-linguistic question.

### List of changes proposed in this PR (pull-request)
- 

### What should a reviewer concentrate their feedback on?
- 

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: ACCESSIBILITY.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+We aim to draw from the [access-centered framework](https://accesscenteredmovement.com/what-access-centered-means/) in describing access as a constant process that changes in each space with each individual. Therefore, this policy focuses both on accessibility as it relates to disabled people as well as practices that enable broader participation in and contribution to The Turing Way book project as well as the wider community. 
+
+[ Add more here about the specific access points of the project ] 
+
+This policy is a work in progress, as are the practices that we employ as a community. Much like how The Turing Way itself is always being written and rewritten, our accessibility policies and practices are constantly being updated as broader awareness grows and changes in technology are made.
+
+Core Principles
+[ Note: these are not being addressed yet, as we work through the the broader policy first! ]
```
Comment (SaranjeetKaur) [inline ACCESSIBILITY.md:12]:
```suggestion
[ Note: these are not being addressed yet, as we work through the broader policy first! ]
```

File: ACCESSIBILITY.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+1. We do these accessibility practices to remove barriers for people who historically have been limited in their ability to be able to participate because these projects benefit everyone
+2. Openness is a means to enable people with broad backgrounds to get involved. Openness for openness' sake does that erase this fact.
+3. Access to commons is a question of mangagmenet - developing a collective commons is
```
Comment (SaranjeetKaur) [inline ACCESSIBILITY.md:16]:
```suggestion
3. Access to commons is a question of management - developing a collective commons is 
```

**Comment by kallewesterling** [comment]
Looks like the CI fails:

```
/home/runner/work/the-turing-way/the-turing-way/book/website/community-handbook/accessibility.md: WARNING: document isn't included in any toctree
```

Solution: **The accessibility.md file needs to be added to the `book/website/_toc.yml` file.**

**Comment by LizHareDogs** [comment]
The edits to the policy document make it much clearer-- thank you all!

File: ACCESSIBILITY.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+This policy is a work in progress, as are the practices that we employ as a community. Much like how _The Turing Way_ itself is always being written and rewritten, our accessibility policies and practices are constantly being updated as broader awareness grows and changes in technology are made.
+
+## Core Principles
+1. We do these accessibility practices to remove barriers for people who historically have been limited in their ability to participate in because these projects benefit everyone
+2. Openness is a means to enable people with broad backgrounds to get involved. Openness for openness' sake does that erase this fact.
+3. Access to commons is a question of mangagmenet - developing a collective commons is 
+
+## How we work with Access-related Feedback
+Access-related feedback may be submitted via this anonymised form, which .
```
Comment (AlexandraAAJ) [inline ACCESSIBILITY.md:19]: seems like there's a part missing after which.

**Review by AlexandraAAJ — COMMENTED** [review]
I have left a comment of a line that seems to be missing information.

File: ACCESSIBILITY.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+This means that questions of access are tied to how people read the guides as well as their ability to participate the project and the community more broadly. Doing so requires navigating a number of different platforms (such as Github, Slack, and others) and languages that may be (). In each of these practices, we aim to facilitate access in a broad sense.
+
+This policy is a work in progress, as are the practices that we employ as a community. Much like how _The Turing Way_ itself is always being written and rewritten, our accessibility policies and practices are constantly being updated as broader awareness grows and changes in technology are made.
+
+## Core Principles
+1. We do these accessibility practices to remove barriers for people who historically have been limited in their ability to participate in because these projects benefit everyone
+2. Openness is a means to enable people with broad backgrounds to get involved. Openness for openness' sake does that erase this fact.
+3. Access to commons is a question of mangagmenet - developing a collective commons is 
+
```
Comment (aleesteele) [inline ACCESSIBILITY.md:17]:
```suggestion

# Guidance for writing/drafting chapters 
[ Guidance for writing in an access-centered way ] 



```

**Comment by aleesteele** [comment]
Connecting this to #3427, where practices from the TTW Community Handbook are being documented.

In the Accessibility Working Group (@the-turing-way/accessibility-working-group), we have separated the Accessibility Policy (ACCESSIBILITY.md) from the other documentation efforts within the project:

1. Accessibility Policy/ACCESSIBILITY.md: defining the standards that we hold ourselves to in the TTW community
2. Community Handbook: Documenting what practices we **currently** do within the community to enable broader access
3. Accessibility Guide: Creating a general guide for access-related practices that may or may not be currently used within the community (tied to #3307)

So they're all connected, but can be written at different speeds. I'm working on the Community Handbook documentation, as it enables me to add what we're doing right **now**, perhaps as opposed to or preparing for what we would like to do in the future.

**Comment by aleesteele** [comment]
Closing this - and fixing this branch!

File: ACCESSIBILITY.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+We aim to draw from the [access-centered framework](https://accesscenteredmovement.com/what-access-centered-means/) in describing access as a constant process that changes in each space with each individual. Therefore, this policy focuses both on accessibility as it relates to disabled people as well as practices that enable broader participation in and contribution to The Turing Way book project as well as the wider community. 
+
+This means that questions of access are tied to how people read the guides as well as their ability to participate the project and the community more broadly. Doing so requires navigating a number of different platforms (such as Github, Slack, and others) and languages that may be (). In each of these practices, we aim to facilitate access in a broad sense.
```
Comment (RichardJActon) [inline ACCESSIBILITY.md:9]:
```suggestion
This means that questions of access are tied to how people read the guides as well as their ability to participate in the project and the community more broadly.
Doing so requires navigating a number of different platforms (such as Github, Slack, and others) and languages that may be ().
In each of these practices, we aim to facilitate access in a broad sense.
```

File: ACCESSIBILITY.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+3. Direct email to RCM with feedback for 1:1
+4. Drop by Accessibility Working Group meeting -> can focus on feedback
+
+Feedback is taken to monthly meeting with chairs of working groups & project leadership.
+
+## Core Principles
+1. We do these accessibility practices to remove barriers for people who historically have been limited in their ability to participate in because these projects benefit everyone
+2. Openness is a means to enable people with broad backgrounds to get involved. Openness for openness' sake does that erase this fact.
+3. Access to commons is a question of management - developing a collective commons is 
+
```
Comment (RichardJActon) [inline ACCESSIBILITY.md:56]:
```suggestion
```

Duplicated from above, see line 13

File: book/website/community-handbook/accessibility/accessibility-alt-text.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
+(ch-accessibility-alttext)=
+# Alt text
+
+Alternative text (or alt text) are used for describing the appearance and function of an image on an HTML page for users who cannot see it.
```
Comment (RichardJActon) [inline book/website/community-handbook/accessibility/accessibility-alt-text.md:4]:
```suggestion
Alternative text (or alt text) is used for describing the appearance and function of an image on an HTML page for users who cannot see it.
```

File: book/website/community-handbook/accessibility/accessibility-alt-text.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+
+
+## Best practices for describing an image in an alt text
+
+Writing alt text is all about context. 
+Being aware of how the image is presented and the context in which it sits will give any assistive technology user a better experience. 
+If you’re writing about an image of a painting, you might want to consider if the style of painting is important, does knowing the painter’s name add value? It’s up to you to decide what information is essential.
+
+Here are a few things to keep in mind when writing alt text:
+* Alt text should be specific and not overly descriptive. They should convey the main point of the image, such as why the image was included or what the author wants the reader to notice
```
Comment (RichardJActon) [inline book/website/community-handbook/accessibility/accessibility-alt-text.md:18]:
```suggestion
* Alt text should be specific and not overly descriptive. 
  It should convey the main point of the image, such as why the image was included or what the author wants the reader to notice
```

File: book/website/community-handbook/accessibility/accessibility-alt-text.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+
+**However**, this formatting does not allow images to be responsive to screen sizes, making them inaccessible to read on small screens and smartphones.
+Furthermore, this doesn't allow authors to resize figures in their chapters or cross-reference them somewhere else in the book.
+
+Therefore, our recommendation is to use [Markedly Structured Text](https://myst-parser.readthedocs.io/en/latest/) (MyST) format available in Jupyter Book.
+
+You can resize figures to adjust how they appear in our chapters using the parameters: `width` and `height` (takes value in px, for example, 400px) or `scale` (takes value in percentage, for example, 50%), especially if your original figure is large.
+Using the parameter: `name`, you can reference figures in other chapters in a similar manner as defined in {ref}`ch-style-crossref`.
+
+**The example figure we have use here can be explained with this alt text:**
```
Comment (RichardJActon) [inline book/website/community-handbook/accessibility/accessibility-alt-text.md:46]:
```suggestion
**The example figure we have used here can be explained with this alt text:**
```

**Comment by aleesteele** [comment]
@RichardJActon - we are pulling over your edits to this updated pull request, thank you! #3581

File: ACCESSIBILITY.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+2. Openness is a means to enable people with broad backgrounds to get involved. Openness for openness' sake does that erase this fact.
+3. Access to commons is a question of mangagmenet - developing a collective commons is 
+
+## 2.1 Web Accessibility
+
+[ORGANIZATION OR PROJECT NAME] is committed to ensuring the accessibility of this site for people with disabilities. We pledge to meet [W3C WAI's Web Content Accessibility Guidelines 2.0](https://www.w3.org/TR/WCAG/), Level AA conformance. Any issues should be reported to [EMAIL ADDRESS].
+
+We are committed to ensuring accessibility through:
+
+- Use of ARIA landmark roles.
```
[the remark this answers is not in the document]
Comment (aleesteele) [reply ACCESSIBILITY.md:24]: This section of specific practices was taken out (in order to keep the practices as broad as possible) - and will need to be added in and linked to in the guides!

File: ACCESSIBILITY.md (hunk 6 above)
Comment (aleesteele) [reply ACCESSIBILITY.md:56]: Wow - this was an old version! Glad you got to see this - but we've updated the principles according to feedback to: Removing barriers, Sharing openly, and Supporting inclusive participation

File: book/website/community-handbook/accessibility/accessibility-alt-text.md (hunk 7 above)
Comment (aleesteele) [reply book/website/community-handbook/accessibility/accessibility-alt-text.md:4]: This is a great edit - we'll add to our alt text page!

**Pull request closed without merging**